### PR TITLE
6.1.1: Add release notes

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds release notes for 6.1.1

## Motivation and Context
Support 6.1.1 release

### CODE REVIEW

File: content/sensu-go/6.1/release-notes.md [hunk 1]
```diff
@@ -70,6 +71,29 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+**FIXES:**
+
+- ([Commercial feature][172]) [`sensuctl prune`][192] now properly handles hooks when pruning resources.
+- ([Commercial feature][172]) Fixed a bug that returned incorrect `!=` results for label selectors when no labels were defined.
+- ([Commercial feature][172]) In the web UI, fixed a bug that could cause a GraphQL `no claims` error when a user's access token was no longer valid instead of displaying the sign-out dialog window.
+- ([Commercial feature][172]) In the web UI, form validation for subscription names now matches allowed values.
+- Fixed a bug that could cause silences to take too long to expire if the cluster goes through a restart or an etcd election.
+- Fixed a bug that prevented sensu-agent from shutting down correctly.
+- Per-entity subscriptions now persist with PATCH requests.
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/release-notes.md:90]:
```suggestion
- Entities are now properly created using `sensuctl create`.
- Per-entity subscriptions now persist with PATCH requests.
```

File: content/sensu-go/6.1/release-notes.md [hunk 2]
```diff
@@ -70,6 +71,29 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.1.1 release notes
+
+**October XX, 2020** &mdash; The latest release of Sensu Go, version 6.1.1, is now available for download.
+
+This patch release includes a number of bug fixes for `sensuctl prune` and `sensuctl dump`, subscription name validation in the web UI, permissions for PATCH requests, and silence expiration times.
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/release-notes.md]:
```suggestion
This patch release includes a number of bug fixes that affect proper hook handling with `sensuctl prune` and `sensuctl dump`, entity creation via `sensuctl create`, form validation for subscription names in the web UI, permissions for PATCH requests, and silence expiration times, among others.
```

File: content/sensu-go/6.1/release-notes.md [hunk 3]
```diff
@@ -70,6 +71,30 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.1.1 release notes
+
+**October XX, 2020** &mdash; The latest release of Sensu Go, version 6.1.1, is now available for download.
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/release-notes.md]:
```suggestion
**October 22, 2020** &mdash; The latest release of Sensu Go, version 6.1.1, is now available for download.
```

File: content/sensu-go/6.1/release-notes.md [hunk 4]
```diff
@@ -70,6 +71,30 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.1.1.
+
+**FIXES:**
+
+- ([Commercial feature][172]) [`sensuctl prune`][192] now properly handles hooks when pruning resources.
+- ([Commercial feature][172]) Fixed a bug that returned incorrect `!=` results for label selectors when no labels were defined.
+- ([Commercial feature][172]) In the web UI, fixed a bug that could cause a GraphQL `no claims` error when a user's access token was no longer valid instead of displaying the sign-out dialog window.
+- ([Commercial feature][172]) In the web UI, form validation for subscription names now matches allowed values.
+- Fixed a bug that could cause silences to take too long to expire if the cluster goes through a restart or an etcd election.
```
Comment (palourde) [inline content/sensu-go/6.1/release-notes.md]: This bug fix is actually not included in 6.1.1, we will ship that in 6.2.0!

File: content/sensu-go/6.1/release-notes.md [hunk 5]
```diff
@@ -70,6 +71,30 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- Fixed a bug that prevented sensu-agent from shutting down correctly.
+- Entities are now properly created using `sensuctl create`.
+- Per-entity subscriptions now persist with PATCH requests.
+- Any user with [`update` permissions][190] for a resource can now make PATCH requests for that resource.
+- Sensuctl now resolves symbolic links.
```
Comment (palourde) [inline content/sensu-go/6.1/release-notes.md]: Same as ^, this will only be included in 6.2.0

**Review by palourde — CHANGES_REQUESTED** [review]

File: content/sensu-go/6.1/release-notes.md (hunk 4 above)
Comment (hillaryfraley) [inline content/sensu-go/6.1/release-notes.md]:
```suggestion
```

File: content/sensu-go/6.1/release-notes.md (hunk 5 above)
Comment (hillaryfraley) [inline content/sensu-go/6.1/release-notes.md]:
```suggestion
```

File: content/sensu-go/6.1/release-notes.md [hunk 6]
```diff
@@ -70,6 +71,30 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.1.1 release notes
+
+**October 22, 2020** &mdash; The latest release of Sensu Go, version 6.1.1, is now available for download.
+
+This patch release includes a number of bug fixes that affect proper hook handling with `sensuctl prune` and `sensuctl dump`, entity creation via `sensuctl create`, form validation for subscription names in the web UI, permissions for PATCH requests, and silence expiration times, among others.
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/release-notes.md]:
```suggestion
This patch release includes a number of bug fixes that affect proper hook handling with `sensuctl prune` and `sensuctl dump`, entity creation via `sensuctl create`, form validation for subscription names in the web UI, and permissions for PATCH requests, among other fixes.
```

**Review by palourde — APPROVED** [review]
💯 👍

**Pull request merged**